src/content/web3/index.md

```diff
@@ -1,28 +1,28 @@
 ---
 title: What is Web3 and why is it important?
-description: An introduction to Web3—the next evolution of the internet—and why it matters.
+description: An introduction to Web3—the next evolution of the World Wide Web—and why it matters.
 lang: en
 sidebar: true
 ---
 
 # Introduction to Web3 {#introduction}
 
-Centralization has helped onboard billions of people to the internet and created the stable, robust infrastructure on which it lives. At the same time, a handful of centralized entities have a stronghold on large swathes of the internet, unilaterally deciding what should and should not be allowed.
+Centralization has helped onboard billions of people to the World Wide Web and created the stable, robust infrastructure on which it lives. At the same time, a handful of centralized entities have a stronghold on large swathes of the World Wide Web, unilaterally deciding what should and should not be allowed.
 
-Web3 is the answer to this dilemma. Instead of an internet monopolized by large technology companies, Web3 embraces decentralization and is being built, operated, and owned by its users. Web3 puts power in the hands of individuals rather than corporations.
+Web3 is the answer to this dilemma. Instead of a Web monopolized by large technology companies, Web3 embraces decentralization and is being built, operated, and owned by its users. Web3 puts power in the hands of individuals rather than corporations.
```

Comment:
```suggestion
Web3 is the answer to this dilemma. Instead of an internet monopolized by large technology companies, Web3 embraces decentralization and is being built, operated, and owned by its users. Web3 puts power in the hands of individuals rather than corporations.
```

Reply:
Same comment as above.